assets/js/components/wp-dashboard/WPDashboardIdeaHub.js

```diff
@@ -63,12 +70,36 @@ function WPDashboardIdeaHub() {
 		}
 	);
 
+	const [ trackingRef, inView ] = useInView( {
+		triggerOnce: true,
+		threshold: 0.25,
+	} );
+
+	useEffect( () => {
+		if ( inView ) {
+			trackEvent(
+				'idea_hub_wpdashboard',
+				'savedposts_notification_view'
+			);
+		}
+	}, [ inView ] );
```

Comment:
Same as above – I think we should ensure this is only tracked once per page view.

Reply:
@aaemnnosttv we use `triggerOnce: true` param in the `useInView` hook to make sure this is triggered only once.

Reply:
Ah thanks, I missed that part 👍